source/WindowPeek.cpp

```diff
@@ -353,17 +354,32 @@ WindowPeek::WindowPeek( BRect R, char* name , Setup* s, Language *w)
   BRect *newFrame = new BRect( 0, top, right, Bounds().Height());
      filePane = new ViewFile( *newFrame , "fileFrame", "/boot/home" , setup, words);
```

Comment:
When using filePane in a layout, ViewFile() should also be LM aware. The BRect newFRame won't be needed any longer (same below in line 356)

Reply:
Yes, I know that more cleaning of that part is needed.

Reply:
> When using filePane in a layout, ViewFile() should also be LM aware. The BRect newFRame won't be needed any longer (same below in line 356)

Using LM-aware constructors results in a blank window. It seems that `filePane` and `imageView` requires a window to initialize its views, then it can resize according to the layout.

I know the numerous `AddChild`s after setting the layout is non-standard, however, omitting them results in an empty window.

Using a `BGroupBuilder` results in crashes in each `AddChild` after the layout is created. And removing the problematic `AddChild`s, like I said before, results in a blank window.

In spite of the weird usage of Layout Management tools, the implementation still works. Like in the [screenshot](https://user-images.githubusercontent.com/57174311/72369964-c53a1800-3733-11ea-84f1-eaf77e1060c8.png) above, the file pane does not conform to the values of the `BRect` passed to the constructor, and the image view is exactly 5.0 times as wide as the file pane.